lib/screens/create_region_screens/components/create_region_confirmation_dialog.dart

```diff
@@ -17,7 +19,10 @@ class CreateRegionConfirmationDialog extends StatelessWidget {
         icon: const CustomPaint(size: Size(60, 60), painter: RedExclamationCircle()),
         leftButtonTitle: "Cancel",
         rightButtonTitle: "Yes I'm sure",
-        onRightButtonPressed: () {},
+        onRightButtonPressed: () {
+          BlocProvider.of<CreateRegionBloc>(context).add(const OnConfirmCreateRegionTapped());
```

Comment:
what happens when we fire an event then navigate away? 
Oh wait, this should load, until the transaction finishes. 

I guess follow up prs

Reply:
yes correct, Once the region is created it should navigate to that region or show a message if fails